hack/make-rules/test-cmd.sh

```diff
@@ -276,2703 +122,16 @@ else
   }
 }
 __EOF__
-fi
-
-# Expose kubectl directly for readability
-PATH="${KUBE_OUTPUT_HOSTBIN}":$PATH
-
-kube::log::status "Checking kubectl version"
-kubectl version
-
-# TODO: we need to note down the current default namespace and set back to this
-# namespace after the tests are done.
-kubectl config view
-CONTEXT="test"
-kubectl config set-context "${CONTEXT}"
-kubectl config use-context "${CONTEXT}"
-
-i=0
-create_and_use_new_namespace() {
-  i=$(($i+1))
-  kubectl create namespace "namespace${i}"
-  kubectl config set-context "${CONTEXT}" --namespace="namespace${i}"
-}
-
-runTests() {
```

Comment:
I had added it long back when we had both "v1" and "v1beta1". We always pass "v1" everywhere now so its obsolete.